CI:

- https://github.com/ClickHouse/ClickHouse/issues/87653
- docker hub issues

```
E               Error response from daemon: Get "https://registry-1.docker.io/v2/": net/http: request canceled while waiting for connection (Client.Timeout exceeded while awaiting headers)
```

--- a/tests/integration/test_keeper_memory_soft_limit/test.py
+++ b/tests/integration/test_keeper_memory_soft_limit/test.py
@@ -58,7 +58,7 @@ def test_soft_limit_create(started_cluster):
         txn.create("/test_soft_limit/node_1000001" + str(i), b"abcde")
         txn.commit()
         node.query("system flush logs metric_log")
-        assert "0\n"  == node.query("select sum(ProfileEvent_ZooKeeperHardwareExceptions) from system.metric_log")
+        assert int(node.query("select sum(ProfileEvent_ZooKeeperHardwareExceptions) from system.metric_log").strip()) > 0
         return
 
     raise Exception("all records are inserted but no error occurs")

